src/controllers/controllerinputmappingtablemodel.cpp

```diff
@@ -197,6 +197,17 @@ QVariant ControllerInputMappingTableModel::data(const QModelIndex& index,
         }
 
         const MidiInputMapping& mapping = m_midiInputMappings.at(row);
+
+        if (std::holds_alternative<std::shared_ptr<QJSValue>>(mapping.control)) {
+            return QVariant();
+        }
+
+        const auto* const control = std::get_if<ConfigKey>(&mapping.control);
+
+        VERIFY_OR_DEBUG_ASSERT(control != nullptr) {
+            return QVariant();
+        }
```

Comment:
this can be simplified, the first if can be removed.

Reply:
IU don't think this should `VERIFY_OR_DEBUG_ASSERT` in line 201. This is a valid case, not a code error. Originally, lines 207-209 weren't present but I  think I remeber I wa told to replace `std::get` by `std::get_if` and add `VERIFY_OR_DEBUG_ASSERT`.

But 201-209 can perfectly be replaced by:

```suggestion
        if (std::holds_alternative<std::shared_ptr<QJSValue>>(mapping.control)) {
            return QVariant();
        }

        const auto* const control = std::get<ConfigKey>(&mapping.control);
```

Reply:
Yup, I'm sorry the current state is fine, I totally looked past the `VERIFY_OR_DEBUG_ASSERT`.